src/lib/components/event/event-summary-row.svelte

```diff
@@ -46,7 +45,7 @@
   export let expandAll = false;
   export let typedError = false;
   export let active = false;
-  export let onRowClick: () => void = noop;
+  export let onRowClick: () => void = () => {};
```

Comment:
These look goo to me but this function could also be `() => void` no need to change just commenting.

Reply:
I thought `void` was only valid as a type declaration, not the function itself 🤔